cosmos/models/group_member.py

```diff
@@ -0,0 +1,20 @@
+from django.contrib.auth.models import Group, User
+from django.db import models
+
+ROLES = [
+    "General Member",
+    "Chairperson",
```

Comment:
Just "Chair" sounds nicer to me than "Chairperson", but im not sure what other people think.

Reply:
sure thing, I was just adapting the old code which had "Chairman"